spec/spec_helper_acceptance.rb

```diff
@@ -12,6 +12,9 @@
   # Configure all nodes in nodeset
   c.before :suite do
     install_module
-    install_module_dependencies
+    # install_module_dependencies
+    install_module_from_forge('puppetlabs/stdlib',  '>= 4.25.0 < 6.0.0')
```

Comment:
Hopefully a new release of camptocamp/postfix can be arranged.  This would include https://github.com/camptocamp/puppet-postfix/pull/246 (allow stdlib 6.x).

Reply:
so i guess this PR is blocked on https://github.com/camptocamp/puppet-postfix/issues/266 then.

Reply:
Nope.  The new release landed this morning.  I just need to tidy a few things and I think we're good to go.  Does it work for you with Debian packages?  Where do they come from?  Is this something we could add to the acceptance tests?

Reply:
oh cool. i didn't have the time to test this against my environment but assuming you started from my code, yes, it does work with the Debian packages, which come from Debian itself. and yes, it's certainly something that could be added to the tests, but i don't see this as a blocker.